include/Core/Base/Mixin/Renderer.php

```diff
@@ -111,4 +111,54 @@ protected function render_user_role_dropdown() {
 	protected function render_post_type_dropdown() {
 		bd_render_post_type_dropdown( $this->field_slug );
 	}
+
+	/**
+	 * Render Taxonomy dropdown.
+	 */
+	protected function render_taxonomy_dropdown() {
+		$taxonomies = get_taxonomies( array(), 'objects' );
+		?>
+		<select name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_taxonomy" class="select2" multiple="multiple" data-placeholder="<?php _e( 'Select Taxonomy', 'bulk-delete' ); ?>">
```

Comment:
Avoid using `select2` in class names. Know more on why you should avoid it [here](https://github.com/sudar/bulk-delete/pull/365#issue-199372401).